blocks/content-api-source-block/sources/content-api-gallery.js

```diff
@@ -0,0 +1,8 @@
+import { fetch, params } from "./content-api";
+
+export default {
+	fetch,
+	params,
+	schemaName: "ans-item",
+	searchable: "gallery",
+};
```

Comment:
Couldn't this work here?
```suggestion
import contentAPI from "./content-api";

export default {
	...contentAPI
	searchable: "gallery",
};
```

So there's no need to add the `export` keywords in the main `content-api.js`.

Reply:
@vgalatro yeah that works as well, went ahead and updated it